internal/kafka/internal/api/public/model_kafka_request.go

```diff
@@ -33,6 +33,7 @@ type KafkaRequest struct {
 	FailedReason                string    `json:"failed_reason,omitempty"`
 	Version                     string    `json:"version,omitempty"`
 	InstanceType                string    `json:"instance_type,omitempty"`
+	DisplayName                 string    `json:"display_name,omitempty"`
```

Comment:
Why is `display_name` part of the request?

Reply:
its a part of the response actually. the naming isn't the best in this case